Cannot build with tsc

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Found 35 errors in 1 files.

Errors  Files
    35  node_modules/.pnpm/puppeteer@13.7.0/node_modules/puppeteer/lib/types.d.ts:1238
    
    Few Error lines are as below,
    node_modules/.pnpm/@types+puppeteer@5.4.6/node_modules/@types/puppeteer/index.d.ts:55:46 - error TS2304: Cannot find name 'Element'.

55 export type WrapElementHandle<X> = X extends Element ? ElementHandle<X> : X;
                                                ~~~~~~~

node_modules/.pnpm/@types+puppeteer@5.4.6/node_modules/@types/puppeteer/index.d.ts:89:56 - error TS2304: Cannot find name 'Element'.

89     $eval<R>(selector: string, pageFunction: (element: Element) => R | Promise<R>): Promise<WrapElementHandle<R>>;

On dev nodemon server its working. But cannot build project.
Issue started after cleaning up node_modeules folder and reinstalling packages.
 
    

### Expected behavior

I expect to build the project and compile ts to js.

But typescript compilation error.

I think some type issues with puppeteer@13.7.0

### Steps to Reproduce the Bug or Issue

1. Remove node_modules
2. Reinstall npm packages, (npm i, yarn, pnpm i)
3. Run "tsc" to build the typescript project
4. You will see types error.

### Relevant Code

_No response_

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

OS: Ubuntu 20
Phone OS: Android
whatsapp-web.js version : 1.17.1
WhatsApp Web version : 2.2230.10
Node.js Version : v18.7.0

### Additional context

_No response_

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,7 +1,7 @@
 
 import { EventEmitter } from 'events'
 import { RequestInit } from 'node-fetch'
-import puppeteer from 'puppeteer'
+import * as puppeteer from 'puppeteer'
 
 declare namespace WAWebJS {
 
@@ -399,10 +399,10 @@ declare namespace WAWebJS {
      * Remote store interface
      */
     export interface Store {
-        sessionExists: ({session: string}) => Promise<boolean> | boolean,
-        delete: ({session: string}) => Promise<any> | any,
-        save: ({session: string}) => Promise<any> | any,
-        extract: ({session: string, path: string}) => Promise<any> | any,
+        sessionExists: (options: { session: string }) => Promise<boolean> | boolean,
+        delete: (options: { session: string }) => Promise<any> | any,
+        save: (options: { session: string }) => Promise<any> | any,
+        extract: (options: { session: string, path: string }) => Promise<any> | any,
     }
 
     /**

